### DISCUSSION TITLE
Javelins not locking.

### DISCUSSION THREAD

**Discussion by coltKeegan** [official-antistasi-community/A3-Antistasi#3124] [Q&A]
Hi, 
So I'm noticing since a couple of weeks that after the update, Javelin missiles are not locking on to any vehicle (land/aerial)....tried changing the locking on keybind and every other possible fix but it simply does not lock on, the only mode I'm able to use is the laser guided tracing mechanism....any help would  be appreciated greatly!

**Comment by MarkTwoFive** [comment]
Hi, I've just been using Javelins ingame and it works just fine for me. Has this been fixed for you too?

**Comment by blistik** [comment]
you need a minimum of 100m to the target before a lock will occur
